time tiempoEjecucion
int32 anguloPitch
int32 anguloRoll
int32 anguloYaw
int32 velocidadPitch
int32 velocidadRoll
int32 velocidadYaw
int32 altura
int32 encendido
int64 mensajesRecibidos
